### PR TITLE
Add brew support

### PR DESCRIPTION
**Pull request by allcloud-jonathan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,17 @@ perform the following:
...
+### Install using Homebrew
+
+To install using Homebrew run the following commands:
+
+    brew tap allcloud-io/tools
+    brew install allcloud-io/tools/clisso
+
+To update just run:
+
+    brew upgrade allcloud-io/tools/clisso
```
Comment (johananl) [inline README.md]: Again, `brew upgrade clisso` is better IMO.

**Comment by allcloud-jonathan** [comment]
Technically `brew install clisso` will work as long as there is no clisso in the core repo. Same is true for upgrade.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Clisso uses the [SAML][7] standard to authenticate users.
 
 ## Installation
 
-### Using a Pre-Compiled Binary
+### Using A Pre-Compiled Binary
```
Comment (johananl) [inline README.md]: Actually the "a" should be lower case... sorry :-)
http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html

**Comment by johananl** [comment]
At the moment there is no `clisso` formula in the core repo, and I plan to have *our* `clisso` there first ;-)

**Pull request merged**